.github/stale.yml

```diff
@@ -1,14 +1,23 @@
 # Number of days of inactivity before an issue becomes stale
 daysUntilStale: 30
+
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 7
-any-of-issue-labels: unconfirmed,cant-reproduce,stale
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: [unconfirmed,cant-reproduce]
```

Comment:
>we want (...) to apply stale if the issue has any of those labels not necessarily all.

I did hesitate when reading the comment as well...

@stumpylog Are you more familiar with this app? Does it actually work that way?

<br>

>I suppose we can just consolidate the labels to or just use unconfirmed only.

The two labels are kinda similar anyway, no?

Reply:
I'm not sure either.  It's kind of a weird way to configure it if that is how it works.  I guess we could leave it only for `cant-reproduce`, since that's generally for something we've tried to reproduce, but can't, while `unconfirmed` is more like getting information still, trying to reproduce?

Reply:
That’s a good idea, just use `cant-reproduce` for stale and unconfirmed is more like pre-triage.